k2/csrc/fsa_util.cc

```diff
@@ -0,0 +1,43 @@
+// k2/csrc/fsa_util.cc
+
+// Copyright (c)  2020  Fangjun Kuang (csukuangfj@gmail.com)
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+#include "k2/csrc/fsa_util.h"
+
+#include <utility>
+#include <vector>
+
+namespace k2 {
+
+void GetEnteringArcs(const Fsa &fsa, VecOfVec *entering_arcs) {
+  // CHECK(CheckProperties(fsa, KTopSorted));
+
+  int num_states = fsa.NumStates();
+  std::vector<std::vector<std::pair<Label, StateId>>> vec(num_states);
+  int num_arcs = 0;
```

Comment:
int32_t

Reply:
thanks, will change it in the next commit.

Reply:
Done.